### DISCUSSION TITLE
SEF URL field as combination of 2 fields in Admin UI - how to do it?

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5085] [Q&A Plugins]
Is it possible to generate sef-url value as combination of 2 fields?

This is from one field:
`'writeParms' => array('size' => 'xxlarge&sef=topictext', 'sef' => 'topictext'),`

Or is it possible without admin UI at all? Just in normal form? What is trigger for this or something like that?

Thanks

**Comment by 3l3ktr0n** [comment]
I guess just combine the 2 strings and after title2sef the variable

**Reply by Jimmi08** [reply]
Thanks for your answer.  Not sure what you mean :) 

I can do this in after update  method, or in the field with type "method".   What I don't know is how to get the same behavior as in normal sef field.   But I didn't get to this in deep...   for now I just disabled that field  - I don't allow to edit it - and it is generated and overwritten when saving the record.
